.rubocop.yml

```diff
@@ -28,6 +28,16 @@ Metrics/LineLength:
   Max: 100
   Exclude:
     - Gemfile
-
+  
 Style/Documentation:
   Enabled: false
+
+
+Style/FormatString:
+  Enabled: false
+Style/DotPosition:
+  Enabled: false
+Style/TrailingWhitespace:
+  Enabled: false
```

Comment:
This is exactly why I'd love us to have a `.rubocop.yml` that everyone uses ^_^
I only added those things so that Code Climate would shut up on things I consider non-issues. The former isn't mentioned in our guidelines; the latter is, so I'll fix that — in a separate PR to keep this clean though.